src/main/java/com/autilus/hw2706/controller/ConsoleHandler.java

```diff
@@ -0,0 +1,39 @@
+package com.autilus.hw2706.controller;
+
+import com.autilus.hw2706.dao.DaoInterface;
+import com.autilus.hw2706.lib.Inject;
+import com.autilus.hw2706.model.Bet;
+import com.autilus.hw2706.model.User;
+
+import java.util.Scanner;
+
+public class ConsoleHandler {
+    @Inject
+    private static DaoInterface<User> userDao;
```

Comment:
cs

Reply:
we need add a blank line before annotation

Reply:
done